packages/docusaurus/src/webpack/plugins/ChunkManifestPlugin.js

```diff
@@ -50,8 +50,11 @@ class ChunkManifestPlugin {
       chunkManifest = assetsMap;
       if (!this.options.inlineManifest) {
         const finalPath = path.resolve(outputPath, this.options.filename);
-        fs.ensureDirSync(path.dirname(finalPath));
-        fs.writeFileSync(finalPath, JSON.stringify(chunkManifest, null, 2));
+        fs.ensureDir(path.dirname(finalPath), () => {
+          fs.writeFile(finalPath, JSON.stringify(chunkManifest, null, 2), done);
+        });
+      } else {
```

Comment:
cannot remove redundant else because need to call done callback.

Reply:
It's not a redundant else if there's no `return` in the `if` block.